common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -821,15 +821,22 @@ StScrollBar {
     background-color: transparent;
     border: none;
 
-    .panel-button {
-      color: $fg_color;
-      &:hover { color: $fg_color; }
+    .panel-button,
+    .panel-button.clock-display .clock {
```

Comment:
Sorry, I didn't bother to test this earlier, and apparently my suggestion didn't didn't work.

The panel `.clock` has to be styled specifically for the `&:focus, &:active, &:checked` `.panel-button`, e.g. with the following ruleset. I couldn't think of a simpler solution, but hopefully this can be cleaned up later.

```
.panel-button {
  &:focus, &:active, &:checked {
    &, &.clock-display .clock {
      color: $selected_fg_color;
      border-color: transparent;
    }
  }
}
```

Reply:
Ok